### PR TITLE
Improvement/web drive tests

### PR DESCRIPTION
**Pull request by BondarenkoStas**

### CODE REVIEW

File: content/yaml/testing.md [hunk 1]
```diff
@@ -50,5 +50,36 @@ scripts:
 
 ## Flutter drive test
```
Comment (helinanever) [inline content/yaml/testing.md:51]: Drive tests or Driver tests?
I'd also add an introductory sentence saying that Flutter driver tests can be run on emulator or simulator when building for Android or iOS or in a web browser when building for the web.

File: content/yaml/testing.md [hunk 2]
```diff
@@ -50,5 +50,36 @@ scripts:
...
 
-    flutter emulators --launch apple_ios_simulator                  # for android use: flutter emulators --launch emulator
-    flutter drive --target=test_driver/my_drive_target.dart
+#### On a mobile emulator
```
Comment (helinanever) [inline content/yaml/testing.md:53]: I'd rephrase as: Running on a mobile emulator or simulator
This heading is currently ####, but should be ### for consistency.

File: content/yaml/testing.md [hunk 3]
```diff
@@ -50,5 +50,36 @@ scripts:
...
+
+        flutter emulators --launch apple_ios_simulator             # for android use: flutter emulators --launch emulator
+        flutter drive --target=test_driver/my_drive_target.dart
+
+#### Web on a browser
```
Comment (helinanever) [inline content/yaml/testing.md:58]: I'd rephrase as: Running on a web browser
Also, should be ###

File: content/yaml/testing.md [hunk 4]
```diff
@@ -50,5 +50,36 @@ scripts:
...
+        - name: 'Flutter drive web test'
+          script: |
+            sudo safaridriver --enable
+            safaridriver --port 4444 &
+            flutter config --enable-web
+            flutter drive --target=test_driver/button_pressing.dart --browser-name safari --release
+
+
+{{<notebox>}}
+Make sure you use Flutter channel **Beta** or higher since web platform is not available on **Stable**.
```
Comment (helinanever) [inline content/yaml/testing.md:84]: ... since Flutter for web is not yet available on **Stable**.

File: content/yaml/testing.md (hunk 1 above)
Comment (BondarenkoStas) [reply content/yaml/testing.md]: Fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/146/commits/0c31b30a4a008e8d6f708454a8033844adfcb3a3

File: content/yaml/testing.md (hunk 2 above)
Comment (BondarenkoStas) [reply content/yaml/testing.md]: Fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/146/commits/0c31b30a4a008e8d6f708454a8033844adfcb3a3.

Just to stress the point that you can't run web app tests on ios/android emulator, or that you can't run android app in a browser, said it that way.

File: content/yaml/testing.md (hunk 3 above)
Comment (BondarenkoStas) [reply content/yaml/testing.md]: Fixed, plus added that it's not actually run on browser, but browser driver
